packages/trimerge-sync/src/lib/Commits.ts

```diff
@@ -0,0 +1,21 @@
+import {Commit, isMergeCommit} from '../types';
+
+export type CommitRefs = {
+    ref: string;
+    baseRef?: string,
+    mergeRef?: string,
+    mergeBaseRef?: string,
+}
+
+// used for getting any possible refs from a general commit, refs that do not apply will be undefined.
+export function refs(commit: Commit<unknown, unknown>): CommitRefs {
```

Comment:
the name of this doesn't feel like a function, how do you feel about calling it `asCommitRefs`? you could even just return the original object I think??

Reply:
yep, returning commits works, crazy!